components/dashboards-web-component/src/designer/DashboardSettings.jsx

```diff
@@ -75,10 +77,12 @@ class DashboardSettings extends Component {
             .getDashboardByID(this.props.match.params.id)
             .then(response => this.setState({dashboard: response.data}))
             .catch((err) => {
-                if (err.response.status === 401) {
+                if (err.response.status === 403) {
```

Comment:
Shall we use status codes via the following constant file (https://github.com/wso2/carbon-dashboards/blob/master/components/dashboards-web-component/src/utils/Constants.jsx)